IdeaProjects/selenium/src/test/java/PageObject/GoogleSearchPage.java

```diff
@@ -0,0 +1,45 @@
+package PageObject;
+
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class GoogleSearchPage extends PageObject {
+
+    private String baseUrl = "http://www.google.com";
+    @FindBy(id = "lst-ib")
+    private WebElement searchBar;
+
+    @FindBy(id = "resultStats")
+    private WebElement resultsStats;
+
+    private int resultsCount;
+
+
+    public GoogleSearchPage(RemoteWebDriver driver) {
+        super(driver);
+        driver.get(baseUrl);
+    }
+
+
+    public int getResultsCount() {
+        return resultsCount;
```

Comment:
To już kwestia do dyskusji, ja osobiście bym w tej metodzie dopiero wyciągał rezultat ze strony, obcinał stringa i dopiero go zwracał, zamiast zwracać coś co mi inna metoda ustawiła.